### ISSUE TITLE
[Bug][My profile] Link to redmine page is incorrect

### ISSUE THREAD

**Issue by DuyenTTN** [AsianTechInc/AT-Portal#11]
Please check attached picture
![portal](https://cloud.githubusercontent.com/assets/21256519/20334389/47d9e762-abec-11e6-8b8a-84610bec6876.PNG)

**Comment by at-datnm** [comment]
@DuyenTTN 

-Currently, your RedmineID is inputted incorrectly. In near future you will edit profile by yourself, so you can edit your Redmine ID is correctly

Thank for your contribution
ATP-Team

**Comment by at-huydq** [comment]
@at-datnm so wrong data entering? should we close this issue?

**Comment by at-datnm** [comment]
@at-huydq : Yes, We should close this bug. We can edit for @DuyenTTN  or she will edit by herself after we set permission for edit profile

**Comment by at-huydq** [comment]
Please edit the information for her then close this bug then.
